integration-cli/docker_cli_commit_test.go

```diff
@@ -253,6 +253,7 @@ func TestCommitChange(t *testing.T) {
 		"--change", "EXPOSE 8080",
 		"--change", "ENV DEBUG true",
 		"--change", "ENV test 1",
+		"--change", "ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
```

Comment:
I was actually planning to use `/foo`, just kept it like this to keep things working and making things easy to debug if something goes wrong (I don't know how that can happen..). How does `/foo` sound?